torchgen/aoti/fallback_ops.py

```diff
@@ -175,3 +175,12 @@
     "aten.view.dtype": {},
     "aten._weight_int4pack_mm_with_scales_and_zeros.default": {},
 }
+
+# `python torchgen/gen.py --update-aoti-c-shim` will automatically generate
+# c_shim_generic.{h/cpp} based on the list below.
+# Operators in this list are intended to be used in torch/csrc/stable/ops.h
+# Unlike other c_shims, operators in this file do not bypass the dispatcher.
+# The same BC rules apply as inductor_fallback_ops.
+generic_fallback_ops: dict[str, dict[str, list[str]]] = {
```

Comment:
Hehe so we had been hopping on the inductor fallback train, even though we were never using these as fallbacks. Since we're now diverging, i think it is better to name this to just 
```suggestion
aten_shimified_ops: dict[str, dict[str, list[str]]] = {
```

Reply:
cc @desertfire just FYI, we are extending the functionality to codegen generic aten shims here!